lib/didww/client.rb

```diff
@@ -37,6 +37,10 @@ def balance
         Resource::Balance.all.first
       end
 
+      def cdr_exports
+        Resource::CdrExport
```

Comment:
@korobey  I think it was changed by mistake, have nothing in common with Pops

Reply:
Indeed, just moved this to maintain alphabetical order of resources.

Reply:
@korobey  such changes are better to apply via separate commit( to respect git history )

Reply:
@Fivell rebased